src/platform_specific/affinity.win.cc

```diff
@@ -0,0 +1,22 @@
+#ifdef _WIN32
+
+#include <Windows.h>
+#include <cassert.h>
+#include <io.h>
+
+#include "affinity.h"
+#include "utils.h"
+
+typedef DWORD_PTR native_cpu_set;
```

Comment:
Maybe move this inside the `affinity_cores_available` function as `using native_cpu_set = DWORD_PTR;`